src/cpu/jit_uni_pooling.hpp

```diff
@@ -86,6 +86,8 @@ struct jit_uni_pooling_fwd_t: public cpu_primitive_t {
 
     protected:
         virtual status_t set_default_params() override {
+            if (src_pd_.desc()->format == memory_format::any)
```

Comment:
The MKL-DNN logic is that only compute-bound primitives (like (de-)convolution and inner-product) may define the formats they want. All the other primitives (like relu, pulling) should follow whatever a user provides them.
So this line should not appear here...

Reply:
I see, I'll remove that.

Reply:
Done.